.changelog/25845.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:enhancement
+resource/aws_wafv2_rule_group: Add `cookies` attribute
+```
```

Comment:
Also add an entry for the `aws_wafv2_web_acl` resource as the `field_to_match` configuration block is shared.

Reply:
Fixed